app/models/organization.rb

```diff
@@ -143,6 +143,10 @@ class Organization < ApplicationRecord
     end
   end
 
+  def using_lifetime_usage?
+    lifetime_usage_enabled? || progressive_billing_enabled?
```

Comment:
Unfortunately, you can have `progressive_billing` integration without `lifetime_usage`. That's why I'm introducing this new helper. What do you think of the name?